src/framework/app-base.js

```diff
@@ -1471,11 +1471,11 @@ class Application extends EventHandler {
      * @returns {object} A object containing the values calculated to use as width and height.
      */
     resizeCanvas(width, height) {
-        if (!this._allowResize) return; // prevent resizing (e.g. if presenting in VR HMD)
+        if (!this._allowResize) return undefined; // prevent resizing (e.g. if presenting in VR HMD)
```

Comment:
I think these should be null instead of undefined .. as the return type is object.

Reply:
Possibly. I'm mainly just trying to keep the functionality identical and satisfy ESLint.